mailpoet/assets/js/src/webpack_admin_expose.js

```diff
@@ -33,7 +33,12 @@ export * as WordPressUrl from '@wordpress/url';
 export * as WordPressI18n from '@wordpress/i18n';
 export * as WordPressIcons from '@wordpress/icons';
 export * as WordPressDataControls from '@wordpress/data-controls';
+export * as WooCommerceDate from '@woocommerce/date';
+import { DateRangeFilterPicker } from '@woocommerce/components/build';
 
+export const WooCommerceComponents = {
+  DateRangeFilterPicker,
+};
```

Comment:
This is an interesting idea, but at the same time it wouldn't tree-shake well, if Premium had multiple JS chunks or something. What's more noticeable now is that it looks inconsistent with how we export WP components. That said, I think the current method is not ideal, and sometimes I'm wondering how we could improve it. So I guess I don't know, should we unify it? Or... meh?

Reply:
Atm for me "meh" tbh.